runtime/j9vm/javanextvmi.cpp

```diff
@@ -575,6 +575,20 @@ JVM_VirtualThreadHideFrames(JNIEnv *env, jobject vthread, jboolean hide)
 }
 #endif /* JAVA_SPEC_VERSION >= 20 */
 
+#if JAVA_SPEC_VERSION >= 21
+JNIEXPORT void JNICALL
+JVM_VirtualThreadMount(JNIEnv* env, jobject vthread, jboolean hide, jboolean firstMount)
+{
+	/* TODO https://github.com/eclipse-openj9/openj9/issues/16984 */
```

Comment:
I think we should have an implementation here. Looking at VirtualThread.java, it appears that if `hide == true`, this should have the same behavior as `JVM_VirtualThreadMountBegin()` else `JVM_VirtualThreadMountEnd()`. A similar applies for `JVM_VirtualThreadUnmount()`.

Reply:
Updated